benchmarks/logreg_l2/solvers/logistic.jl

```diff
@@ -0,0 +1,33 @@
+
+# Loss evaluation
+function logistic_loss(X, y, w::Array{Float64})
+    # return sum( log.(1. .+  exp.(-(y .* (X*w)))) ) # other implementation
```

Comment:
why did you choose one version vs the other? One is faster? more stable?

Reply:
The uncommented one was used by Robert in StochOpt.jl.
The commented one is mine, according to `@btime` command it allocates less memory than the second and seems to be a bit faster. Let's keep the commented one then.